README.md

```diff
@@ -7,6 +7,16 @@ Common code shared among
 [MySQL](https://github.com/juttle/juttle-mysql-adapter/)
 and [SQLite](https://github.com/juttle/juttle-sqlite-adapter/) adapters.
 
+#### List of optimized operations
+
+* any filter expression `read sql` (note: `read sql | filter ...` is not optimized)
```

Comment:
I'm just not sure if you're talking about this: https://github.com/juttle/juttle-sql-adapter-common/blob/master/test/filters.spec.js#L64

or an actual full regular expression.